apps/hash-graph/lib/graph/src/api/rest/entity.rs

```diff
@@ -459,6 +469,59 @@ impl<'s> ToSchema<'s> for Viewer {
     }
 }
 
+#[derive(Serialize, ToSchema)]
+struct EntityRelationScope {
+    relation: EntityRelation,
+    scope: VisibilityScope,
+}
```

Comment:
Similar to the other comment, I think these are probably better described as `Relationships`... `DirectOwner` is a type of relationship... the user who owns the thing is the relation

Reply:
I changed the naming to match the function.